service/src/main/java/uk/gov/mca/beacons/api/export/xlsx/backup/BackupSpreadsheetRow.java

```diff
@@ -102,20 +100,25 @@ public class BackupSpreadsheetRow implements SpreadsheetRow {
   private String lastModifiedDate;
```

Comment:
I'm confused why we have :

- Domain object
- Export DTO
- Backup Export DTO
- Spreadsheet row

Is there a way we can build the spreadsheet row directly from the domain models?

Reply:
Do you want me to pass in a BeaconBackupItem instead?

Reply:
What's the current flow of the data from the db to the spreadsheet?

Reply:
If you want me to use toLegacyBeaconExportDTO() passing in a LegacyBeacon, won't we have to do a DB call in BackupBeaconToSpreadsheetRowItemProcessor that gets an individual LegacyBeacon?